src/api/spec/components/token_card_component_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe TokenCardComponent, type: :component do
+  let(:user) { create(:user) }
+
+  before do
+    Flipper.enable(:trigger_workflow)
+    User.session = user
+    render_inline(described_class.new(token: token))
+  end
+
+  context 'token with a name' do
+    let(:token) { create(:rebuild_token, user: user, name: 'foo_token') }
```

Comment:
I don't think it's needed to store the token in the database to properly test this view component, so we can rely on `build_stubbed` instead of `create` for all tokens in this file.

Here's an [article](https://thoughtbot.com/blog/use-factory-girls-build-stubbed-for-a-faster-test) on this.

Reply:
right, shouldn't be necessary

Reply:
done